packages/modeling/src/geometries/geom2/toSides.js

```diff
@@ -11,6 +15,13 @@ const applyTransforms = require('./applyTransforms')
  * @example
  * let sharedsides = toSides(geometry)
  */
-const toSides = (geometry) => applyTransforms(geometry).sides
+const toSides = (geometry) => {
+  let sides = cache.get(geometry)
+  if (sides) return sides
+
+  sides = applyTransforms(geometry).sides
+  cache.set(geometry, sides)
```

Comment:
applyTranforms() returns a NEW object every time, so why cache?

Reply:
without caching, if script calls toSides(geometry) multiple times those transformed polygons will be regenerated every time.
this could happen for example
 - during measureBoundingBox first time
 - for booleans or serializeSolids second,third... time

if we have a geometry (translated sphere)
  - toSides will call applyTransforms 
  - applyTransforms generates geometry polygons with transformations applied, and identity transform
  - toSides takes only the polygons and returns them
  - generating a geometry in applyTransforms is pointless


Actually with this PR, applyTransforms becomes obsolete in a way, and has overhead of creating a new geometry that will never be used, only the generated polygons will currently.

Transform from now on just changes transforms matrix and does not use applyTransforms

applyTransforms code could be merged into: toSides, toPoints, etc